libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/hwdef.dat

```diff
@@ -170,6 +170,9 @@ PD14 TIM4_CH3 TIM4 PWM(6) GPIO(55)
 
 define BOARD_PWM_COUNT_DEFAULT 4
 
+# Pin for PWM Voltage Selection
+PB4 PWM_VOLT_SEL OUTPUT GPIO(3)
```

Comment:
normally you would add:
define HAL_GPIO_PWM_VOLT_PIN 3
then you don't need the changes in GPIO.cpp, it can all be done generally